web/src/routes/(user)/people/+page.svelte

```diff
@@ -439,10 +439,10 @@
     {/if}
   </svelte:fragment>
 
-  {#if countVisiblePeople > 0}
+  {#if countVisiblePeople > 0 && (!searchName || (searchName && searchedPeopleLocal.length > 0))}
```

Comment:
```suggestion
  {#if countVisiblePeople > 0 && (!searchName || searchedPeopleLocal.length > 0)}
```
If we specify the condition `searchName` as `x` and `searchedPeopleLocal.length > 0` as `y`, what you have there is `!x || (x && y)`. This however you can factor out to `(!x || x) && (!x || y)`. `!x || x` is tautological, so you can throw that term away resulting in the suggestion :)

Reply:
That's what I call a great optimization :smile: